packages/communication-react/review/communication-react.api.md

```diff
@@ -1019,6 +1019,7 @@ export const COMPOSITE_ONLY_ICONS: {
     LocalPreviewPlaceholder: JSX.Element;
     LocalCameraSwitch: JSX.Element;
     ControlBarButtonBadgeIcon: JSX.Element;
+    MoreDrawerPeople: JSX.Element;
```

Comment:
This name is throwing me off.
Just by reading this, I am not able to figure out what icon this might be

Reply:
Maybe something like `ParticipantDrawerMoreButtonIcon` might be more intuitive?

Reply:
IT'S THROWING ME OFF TOO, I keep picturing drawer people, like small little people who live in a desk drawer and they're popping their head out to say hello 🙋‍♀️

Reply:
Will pick this up in a follow up PR.